roles/install_dbserver/defaults/main.yml

```diff
@@ -3,7 +3,7 @@
 os: ""
 pg_type: "PG"
 pg_version: 14
-bdr_version: 4
+pgd_version: 4
```

Comment:
based on the commit descriptions, shouldn't v5 be the default?

Reply:
This is already resolved in PR.